### PR TITLE
fix the issue to avoid printing out secret information

### PR DESCRIPTION
**Pull request by xinredhat**
It fixed the issues that prints out secret information 

Signed-off-by: Xin Jiang @xinredhat

### CODE REVIEW

**Review by ramessesii2 — APPROVED** [review]
LGTM.
One minor comment - @xinredhat there's no end line at the end of the file and static-check isn't complaining about it. Is it normal?

**Comment by xinredhat** [comment]
> there's no end line at the end of the file and static-check isn't complaining about it. Is it normal?

@ramessesii2 I don't think it's normal, let me double check.

**Review by Roming22 — APPROVED** [review]

**Comment by ramessesii2** [comment]
Hello @xinredhat, I can merge this. Could you please push that newline at the end of file just to maintain consistency

File: ci/images/ci-runner/hack/bin/utils.sh [hunk 1]
```diff
@@ -54,11 +54,11 @@ get_password() {
     setx_off
     local itemid="$1"
     password=$(bw get password "$itemid" --session "$session")
-    echo "$password"
+    export password
     setx_on
 }
 
-get_attachmet(){
+get_attachment(){
```
[the remark this answers is not in the document]
Comment (xinredhat) [reply ci/images/ci-runner/hack/bin/utils.sh]: fixed

**Comment by xinredhat** [comment]
> Hello @xinredhat, I can merge this. Could you please push that newline at the end of file just to maintain consistency

@ramessesii2 Done

**Pull request merged**